Lombiq.BaseTheme/Services/CssClassHolder.cs

```diff
@@ -21,5 +23,12 @@ public void AddClassToZone(string zoneName, string className)
         classes.Add(className);
     }
 
-    public ISet<string> GetZoneClasses(string zoneName) => _zones.GetMaybe(zoneName) ?? new();
+    public ISet<string> GetZoneClasses(string zoneName)
+    {
+        if (_zones.TryGetValue(zoneName, out var zone)) return zone;
```

Comment:
Just like in `AddClassToZone`, the local class collection variable should be called `classes`.

On a similar note, `_zones` should also be called `_zonesWithClasses` or similar.

This might feel annoying now, but we won't come back here any time soon.

Reply:
How about `_classesByZones` or `_classesByZoneNames`? I think naming a dictionary `valuesByKeys` is fairly intuitive and it clearly expresses the relationship between classes and zones.

Reply:
`_classesByZones` sounds like the best option to me.